packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -42,6 +42,17 @@ To run regression tests in the source tree:
 $ test/integ/test-cli-regression-against-current-code.sh [-t '...']
 ```
 
+Integ tests can run in parallel across multiple regions. Set the `AWS_REGIONS`
+environment variable to a comma-separate list of regions:
+
+```
+$ env AWS_REGIONS=us-west-2,us-west-1,eu-central-1,eu-west-2,eu-west-3 test/integ/run-against-repo test/integ/cli/test.sh
+```
+
+No two tests will run in the same region at the same time. Tests parallelize
+up to the concurrency limit imposed by jest (default of 5) and the available
```

Comment:
State minimally on how to control this. 

Clarifying: Is this as simple as changing [`maxConcurrency`](https://jestjs.io/docs/en/cli#--maxconcurrencynum)?  Does this mean all tests right now use `test.concurrent`?

Reply:
Yes